kafka-plugins-0.10/src/main/java/co/cask/hydrator/plugin/sink/KafkaRecordWriter.java

```diff
@@ -44,6 +44,7 @@ protected void sendMessage(final String key, final String body) {
       producer.send(new ProducerRecord<>(topic, key, body)).get();
     } catch (Exception e) {
       LOG.error("Exception while sending data to kafka topic {}, key {}, message {}, e", topic, key, body, e);
+      throw new RuntimeException(e);
```

Comment:
Don't log an exception if it is getting rethrow.

Reply:
Also, why wrap with RuntimeException? The `sendMessage` should be a private method that calls from the `write` method, which allows `IOException` to be thrown. BTW, this KafkaRecordWriter implementation is pretty slow as it doesn't batch writes to Kafka at all.